waspc/data/Generator/templates/sdk/server/email/core/types.ts

```diff
@@ -37,7 +37,7 @@ export type EmailSender = {
 // PRIVATE API
 export type SentMessageInfo = any;
 
-// PRIVATE API
+// PUBLIC API
```

Comment:
Exposes `Email` as public api in `wasp/server/email` @Martinsos do we need to add it to the codemod?

Reply:
So you exposed `Email` and `EmailFromField` in this PR.

I don't think we ever had imports for these defined in the docs before, so, in theory we don't need them in the codemod? But if you think it was likely people used them before nevertheless, despite us not having them mentioned in the docs, then we can add them in the codemod. You can add them yourself every easily, just add the rule on the `main` branch -> if you are not sure about it, let me know and I can also do it or help.

Reply:
Added them in case some people used it 👍